src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -239,12 +239,17 @@ interfaces:
     - series_uuid.series_string=foobar
     sample_value_sets:
       # callingFormCheck: pi_version java: Flattened Request Callable
+      # callingFormCheck: pi_version nodejs: Request
+      # callingFormCheck: pi_version py: Request
     - id: pi_version
       title: "Pi version"
       description: "Testing calling forms: {Java:[Flattened, Request, Callable], NodeJS:[Request], Python:[Request]}"
```

Comment:
Consider also removing the calling form data from the description. We don't need it, keeping it sync willbe in pain, and if it falls out of sync it will be annoying.

Reply:
Let's do this in a separate PR? Enabling checks and removing this are arguably orthogonal.

Reply:
OK, doing it in a chaser will make it easier to focus on the current changes in this PR.